test/bip44_tests.c

```diff
@@ -22,7 +22,7 @@ void test_bip44()
     int result;
     dogecoin_hdnode node;
     dogecoin_hdnode bip44_key;
-    char mnemonic[256] = "";
+    // char mnemonic[256] = "";
```

Comment:
This should probably be deleted if it's not being used

Reply:
Agreed but am not certain if @edtubbs was going to use for something?

Reply:
looks like is fixed in #91